package.json

```diff
@@ -2,21 +2,20 @@
   "name": "courses",
   "version": "1.0.0",
   "private": true,
+  "proxy": "http://localhost:8090",
   "dependencies": {
-    "@material-ui/core": "~3.6.2",
-    "@material-ui/icons": "^3.0.2",
+    "@fullcalendar/daygrid": "^5.1.0",
+    "@fullcalendar/list": "^5.1.0",
+    "@fullcalendar/react": "^5.1.0",
+    "@fullcalendar/timegrid": "^5.1.0",
```

Comment:
Are you using the calendar stuff?  (If not, this can be ripped out.)

Reply:
This has been removed.